README.md

```diff
@@ -13,49 +19,23 @@
 * Objects
 * Errors in Brief
 
-## About
-
-There are two types of data in JavaScript: 1) primitives and 2) objects.
-
-All types except objects define immutable values (values which are incapable of being changed). For example, Strings are immutable. This means that once a string is created, it is not possible to modify it. However, it is still possible to create another string based on an operation on the original string. We refer to values of these types as "primitive values".
+## The Types
 
-## Primitive Data Types
+JavaScript's data types are:
 
-JavaScript has a six primitive data types:
++ Number
++ String
++ Boolean
++ Undefined
++ Null
++ Symbol
 
-1. Number
-  - Examples include: `1`, `3.14`, `0.13`, `100`, ...
-  - The largest number in JavaScript is `9007199254740992`. Similarly, the smallest number is -`9007199254740992` (see [here](http://stackoverflow.com/a/307200/2890716) for more info.
 
-2. String
-  - Examples include:  `"a"`, `"World Wide Web"`, `"Hey! H*w @re y0u?!"`
-
-3. Boolean
-  - Only two examples: `true` and `false`
-`
-4. Undefined
-  - Only one example: `undefined`
-  - Refers to an absent or unknown value. 
-  - A variable that has not been assigned a value has the value undefined.
-  - A function returns undefined if a value is not returned.
-
-5. Null
-  - Only one example: `null`
-  - This is a special keyword that means one of two things: no value or empty.  The difference from undefined is that when a variable is null, it is still defined.
-
-6. Symbol type
- - New (!) to JavaScript in ECMAScript Edition 6. A Symbol is a unique and immutable primitive value and may be used as the key of an Object property. Instead of creating new symbols with the `:` as we do in Ruby, you can make a new symbol by calling on the Symbol class and passing it a string:
-
-```javascript
-var greeting = Symbol("hello");
-typeof sym; // Returns "symbol"
-```
-
-#### 1. Numbers
+### Numbers
 
 Unlike most programming languages JS uses floats by default. And creating literal numbers uses the Number Constructor. New numbers can optionally have decimal points with trailing zeros, but keep in mind they are floats either way.
```

Comment:
I'm confused about the Number Constructor - can "And creating literal numbers uses the Number Constructor." be removed?